frontend/src/components/Layout/Header/SearchCourses/SearchResult.tsx

```diff
@@ -11,19 +13,43 @@ interface CoursesFound {
 
 const LAST_ARRAY_ITEM = -1;
 
-const SearchResult: React.FC<CoursesFound> = ({ coursesFound }) => (
-  <SearchResultWrapper>
-    {coursesFound.length ? (
-      coursesFound.map((course, id, array) => (
-        <SearchResultItemContainer
-          course={course}
-          addDivider={id < array.length + LAST_ARRAY_ITEM}
-        />
-      ))
-    ) : (
-      <NoSearchResults>No results</NoSearchResults>
-    )}
-  </SearchResultWrapper>
-);
+const SearchResult: React.FC<CoursesFound> = ({ coursesFound }) => {
+  const [foundInMyCourses, setFoundInMyCourses] = useState<string | undefined>(undefined);
+
+  const findCourse = (courses: Array<ClientCourse>, title: string): void => {
+    for (let clientCourse = 0; clientCourse < courses.length; clientCourse += 1) {
```

Comment:
`clientCourse` seems like an object, but not number. Rename this variable please

Reply:
done